File: tt/c_cppCode/cheggoct/Player.cpp
//Player class implementation
#include "Player.h"
#include<string>
using namespace std;
//constructor
player::player(){
}
// getters
string player::getFirstName()
{
	return firstName;
}
string player::getLastName()
{
	return lastName;
}
int player::getNumber()
{
	return number;
}
//setters
void player::setFirstName(string fname){
	firstName=fname;
}
void player::setLastName(string lname)
{
	lastName=lname;
}
void player::setNumber(int no)
{
	number=no;
}

File: tt/DS_practice/src/cheggoct/Player.java
package cheggoct;

public class Player {

	//private member declaration
	private String lastName;
	private String firstName;
	private int number;
	//default constructor
	public Player()
	{}
	
	//getters and setters
	public int getNumber() {
		return number;
	}
	public void setNumber(int number) {
		this.number = number;
	}
	public String getFirstName() {
		return firstName;
	}
	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}
	public String getLastName() {
		return lastName;
	}
	public void setLastName(String lastName) {
		this.lastName = lastName;
	}
	//to string method for object string represenation
	public String toString()
	{
		return number+") "+firstName+" "+lastName;
	}
}
